docs/website/root/manual/developer-docs/nodes/mithril-client.md

```diff
@@ -296,20 +296,34 @@ Here is a list of the available parameters:
 | `network` | - | - | `NETWORK` | Cardano network | - | `testnet` or `mainnet` or `devnet` | :heavy_check_mark: |
 | `aggregator_endpoint` | `--aggregator-endpoint` | - | `AGGREGATOR_ENDPOINT` | Aggregator node endpoint | - | `https://aggregator.pre-release-preview.api.mithril.network/aggregator` | :heavy_check_mark: |
 | `genesis_verification_key` | - | - | `GENESIS_VERIFICATION_KEY` | Genesis verification key | - | - | :heavy_check_mark: |
-| `json_output` | `--json` | `-j` | - | Enable JSON output | no | - | - |
+| `log_format_json` | `--log-format-json` | - | - | Enable JSON output logs | no | - | - |
```

Comment:
This is a breaking change as it breaks all the scripts using the old `-j` option.

Reply:
Actually no, the `-j` was an error and didn't exist.